playwright/e2e/crypto/backups.spec.ts

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { type Page } from "@playwright/test";
+
+import { test, expect } from "../../element-web-test";
+
+async function expectBackupVersionToBe(page: Page, version: string) {
+    await expect(page.locator(".mx_SecureBackupPanel_statusList tr:nth-child(5) td")).toHaveText(
+        version + " (Algorithm: m.megolm_backup.v1.curve25519-aes-sha2)",
+    );
+
+    await expect(page.locator(".mx_SecureBackupPanel_statusList tr:nth-child(6) td")).toHaveText(version);
+}
+
+test.describe("Backups", () => {
+    test.use({
+        displayName: "Hanako",
+    });
+
+    test("Create, delete and recreate a keys backup", async ({ page, user, app }, workerInfo) => {
+        // Create a backup
+        const tab = await app.settings.openUserSettings("Security & Privacy");
+
+        await expect(tab.getByRole("heading", { name: "Secure Backup" })).toBeVisible();
+
+        let securityKey = "";
+
+        const currentDialogLocator = page.locator(".mx_Dialog");
+
+        await tab.getByRole("button", { name: "Set up", exact: true }).click();
```

Comment:
Lots of whitespace here, and I don't think it helps readability.

```suggestion
        // Create a backup
        const tab = await app.settings.openUserSettings("Security & Privacy");
        await expect(tab.getByRole("heading", { name: "Secure Backup" })).toBeVisible();
        await tab.getByRole("button", { name: "Set up", exact: true }).click();
```